adam/experiment/observer.py

```diff
@@ -284,14 +341,11 @@ def report(self) -> None:
         pass
 
 
-@attrs(slots=True)
-class HTMLLoggerPostObserver(
-    DescriptionObserver[SituationT, LinguisticDescriptionT, PerceptionT]
-):
+@attrs(slots=True)  # pragma: no cover
+class HTMLLoggerPostObserver(CandidateAccuracyObserver):  # type: ignore
```

Comment:
Prefer composition over inheritance. :-) . `HTMLLoggerPostObserver` should have a `has-a` relation to `CandidateAccuracyObserver`, not `is-a`.